libs/ui-watt/src/lib/components/input/timepicker/watt-timepicker.component.ts

```diff
@@ -43,6 +52,28 @@ import { WattRangeInputService } from '../shared/watt-range-input.service';
 const hoursMinutesFormat = 'HH:MM';
 const hoursMinutesPlaceholder = 'HH:MM';
 
+// Show slider initially as "00:00 - 23:59"
+const initialSliderValue: WattSliderValue = { min: 0, max: 1439 };
+
+/** Converts string time format (HH:MM) to number of minutes. */
+function timeToMinutes(value: string) {
```

Comment:
suggestion: add return type

```suggestion
function timeToMinutes(value: string): number {
```

Reply:
24cc8cbec1c680d32eba28abba117068d85ad83c